syntax = 'proto3';

service NotificationService {
    rpc SendEmail(SendEmailRequest) returns (SendEmailResponse);
}

message SendEmailRequest {
    string recipient = 1;
    string subject = 2;
    string body = 3;
}

message SendEmailResponse {
    bool success = 1;
    string message = 2;
}